crates/revm/src/db/in_memory_db.rs

```diff
@@ -86,26 +100,25 @@ impl<ExtDB: DatabaseRef> CacheDB<ExtDB> {
 // TODO It is currently only committing to cached in-memory DB
 impl<ExtDB: DatabaseRef> DatabaseCommit for CacheDB<ExtDB> {
     fn commit(&mut self, changes: Map<H160, Account>) {
+        // clear storage by setting all values to `0`
+        fn clear_storage(storage: &mut HashMap<U256, U256>) {
+            storage.values_mut().for_each(|val| {
```

Comment:
something to consider in the future, maybe we want to convert these to rayon parallel iters? maybe not worth it

Reply:
For this particular thing, it would be best to introduce storage incarnation: https://github.com/ledgerwatch/erigon/blob/devel/docs/programmers_guide/db_walkthrough.MD#table-plainstate
In general, for introducing rayon, we need to be careful that we have a good amount of data and processing that we want to iterate/Parallelize on.